codegen-sbt/src/sbt-test/codegen/test-compile/build.sbt

```diff
@@ -1,20 +1,24 @@
+import _root_.caliban.tools.Codegen
+
 lazy val root = project
   .in(file("."))
-  .enablePlugins(CodegenPlugin)  // Intentionally maintain the deprecated name
+  .enablePlugins(CalibanPlugin)
   .settings(
     libraryDependencies ++= Seq(
       "com.github.ghostdogpr" %% "caliban-client" % Version.pluginVersion
     ),
     Compile / caliban / calibanSettings ++= Seq(
-      calibanSetting(file("src/main/graphql/schema.graphql"))(  // Explicitly constrain to disambiguate
-        cs =>
-          cs.clientName("Client")
+      calibanSetting(file("src/main/graphql/schema.graphql"))( // Explicitly constrain to disambiguate
```

Comment:
tbh I don't get what this "constrains", is that comment up to date?

Reply:
Certainly not the most helpful comment, as I don't understand what needed to be disambiguated here -- perhaps back when it was possible (or intended) to star-glob multiple files? I could see trying to disambiguate between the few `.graphql` test specifications we had at that time.